embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataSourceModule.kt

```diff
@@ -211,6 +213,18 @@ internal class DataSourceModuleImpl(
         )
     }
 
+    override val rnActionDataSource: DataSourceState<RnActionDataSource> by dataSourceState {
```

Comment:
Should we only create this data source if the framework is react native or is it assumed that any platform can log one of these?

Come to think of it, maybe we should think about moving this to the RN layer - not now, but it feels weird to have to do this in the Android/iOS layer now we know this is just a Span.

Reply:
I agree that we can move this to another layer. But for now I don't see a problem. This is created only after being called, and it's only used by the internal interface.